install-deps.sh

```diff
@@ -344,6 +344,9 @@ else
         case "$ID" in
             fedora)
                 $SUDO dnf install -y dnf-utils
+                $SUDO dnf install -y docker-ce docker-ce-cli containerd.io
```

Comment:
could you please include e33e3a931db97d01318643ec686fe63fdd614082 in this backport?

Reply:
Thanks @tchaikov . Done.